src/components/LoadFileMenu/index.tsx

```diff
@@ -89,24 +93,32 @@ const LoadFileMenu = ({
                 <Button type="ghost" onClick={showModal}>
                     Simularium file
                 </Button>
-            )
+            ),
         },
         {
             key: "file-convert",
             label: (
-                <Button type="ghost" onClick={initializeFileConversion}>
+                <Link
+                    type="ghost"
+                    to={{
+                        pathname: IMPORT_PATHNAME,
+                    }}
+                    onClick={() => {
+                        initializeFileConversion();
```

Comment:
naming: `initializeFileConversion` sounds like it's about to convert something. Consider maybe `initializeFileConversionUI`  or `initializeFileConversionForm`?  kinda nitpicky not a huge deal.

Reply:
got it, went with initializeFileConversionUI